src/hotspot/os/windows/os_windows.cpp

```diff
@@ -3267,21 +3267,22 @@ static char* map_or_reserve_memory_aligned(size_t size, size_t alignment, int fi
   return aligned_base;
 }
 
-char* os::reserve_memory_aligned(size_t size, size_t alignment) {
+char* os::reserve_memory_aligned(size_t size, size_t alignment, bool exec) {
+  // exec can be ignored
```

Comment:
Actually, since exec and non-exec reservations are equal on windows, ignoring exec is a correct implementation. Assert would not fit here. Please let me know if the comment fails to deliver this message. Also, I have a prototype implementation of CDS support for macOS/AArch64 and it needs executable aligned mapping, I think we'll need this parameter anyway.

Reply:
Okay, leave it as it is.